npm/private/pnpm.bzl

```diff
@@ -145,6 +146,10 @@ def _convert_v5_importer_dependency_map(specifiers, deps):
             alias, version = _strip_v5_v6_default_registry(version).lstrip("/").rsplit("/", 1)
             version = _convert_pnpm_v5_version_peer_dep(version)
             version = "npm:{}@{}".format(alias, version)
+        elif version.startswith("link:"):
+            version = version[:5] + paths.normalize(paths.join(import_path, version[5:]))
```

Comment:
bit inconsistent that we hardcode `5` here but `len("link:")` in the file above :)

Reply:
It is inconsistent all over the place atm, maybe that can be cleaned up another time.